### ISSUE TITLE
asurabus Sound Fixes/Clone possible backport

### ISSUE THREAD

**Issue by arnoldsecret** [libretro/mame2003-plus-libretro#911]
0.143u3: hap re-added YMF262 to the FuukiFG3 driver, used in the music for FM channels and fixed sound effects are looping. Brian Troha corrected dipswitches and added DIP locations to Asura Buster. Changed YMF262 clock speed to 14260547 Hz.

The game has a Review build that might be interesting.

http://adb.arcadeitalia.net/dettaglio_mame.php?game_name=asurabusa

**Comment by arcadez2003** [comment]
Yeah i know about these changes from what i mind you can just drop the code from 0.143u3 into the driver you also have to backport some earlier changes leading up to it, i'll look into this again real soon it's the workload required that puts me off.

**Comment by arcadez2003** [comment]
Right this is the commit we need............

https://github.com/arcadez2003/mame/commit/8323f492df8d981c2d1b9ba8bd069b8ddf60bd20#diff-83d91be2e3c59e89b1510c178c850c662844bf2a8b630508c544eeb3a0ea9f73

I might be able to backport the code to this core i wont know till i've looked at it in more depth

**Comment by arcadez2003** [comment]
Ok as i said sometimes you cant just drop in code from a certain MAME build, as for it to slot in correctly you have to 
add previous commits, all these from happ will be required firstly...........

https://github.com/arcadez2003/mame/commits/8323f492df8d981c2d1b9ba8bd069b8ddf60bd20?before=8323f492df8d981c2d1b9ba8bd069b8ddf60bd20+35&branch=8323f492df8d981c2d1b9ba8bd069b8ddf60bd20&path%5B%5D=src&path%5B%5D=emu&path%5B%5D=sound&path%5B%5D=ymf278b.c

**Comment by arcadez2003** [comment]
Im hoping this commit from dink might be enough to stop the repeating echos when it comes to the speech and sfx
in Asura Buster for this core..
https://github.com/SidonCryerd/FinalBurnNeo/commit/6649e7b1d0b7c70c0708eabbb51286c4fee15d63#diff-e11691ea18d978e592320a5e86b46cb153b0966fc68f9612ba6fe185ab1da7c3

This is what i've come up with....
https://github.com/libretro/mame2003-plus-libretro/pull/1819

So @grant2258 @mahoneyt944 if you guys get a spare moment sometime can you take a peek at my PR to see if 
firstly i've implemented this correctly and if so does it stop certain sound repeating/ echoing in Asura Buster

**Comment by arcadez2003** [comment]
https://github.com/SidonCryerd/FinalBurnNeo/blob/6649e7b1d0b7c70c0708eabbb51286c4fee15d63/src/burn/snd/ymf278b.c#L958

**Comment by mahoneyt944** [comment]
I can test the game here shortly, I guess if it sounds ok then this PR should be ok I would think.

**Comment by arcadez2003** [comment]
Thanks man fingers crossed this will be enough to sort Asura Buster sound niggles
